src/components/elements/default-tab.tsx

```diff
@@ -16,6 +16,11 @@ type TabsStyledProps = {
     selected?: boolean
 }
 
+type TextWrapperProps = {
```

Comment:
naming is correct?

Reply:
yes @ali-hosseini-fs as I checked they are using one of these 2 conventions: `ComponentNameProps` || `ComponentNameTypes`